lib/src/gradle_parser.dart

```diff
@@ -0,0 +1,102 @@
+import 'package:petitparser/petitparser.dart';
+
+/// Grammar for Gradle
+class GradleGrammerDefinition extends GrammarDefinition {
```

Comment:
private classes please... Personally I haven't tried to use the `GrammarDefinition` stuff, but I suppose can might make sense.